tests/spec/fs.writeFile-readFile.spec.js

```diff
@@ -99,3 +99,132 @@ describe('fs.writeFile, fs.readFile', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+
+describe('fsPromises.writeFile, fsPromises.readFile', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fsPromises = util.fs().promises;
+    expect(fsPromises.writeFile).to.be.a('function');
+    expect(fsPromises.readFile).to.be.a('function');
+  });
+
+  it('should return a promise', function() {
+    var fsPromises = util.fs().promises;
+    var contents = 'This is a file.';
+
+    expect(fsPromises.writeFile('/myfile', contents)).to.be.a('Promise');
+    expect(fsPromises.readFile('/myfile', 'utf8')).to.be.a('Promise');
+  });
+
+  it('should error when path is wrong to readFile', function(done) {
```

Comment:
Get rid of the `done` callback here, and instead you need to add a `return` to the front of line 128 below, so you return your Promise.